plugin-cgroupv2/src/oar3/plugin.rs

```diff
@@ -188,9 +191,65 @@ impl AlumetPlugin for OARPlugin {
 impl Default for OAR3Config {
     fn default() -> Self {
         let root_path = PathBuf::from("/sys/fs/cgroup/");
+        if !root_path.exists() {
+            log::warn!("Error : Path '{}' not exist.", root_path.display());
+        }
         Self {
             path: root_path,
             poll_interval: Duration::from_secs(1), // 1Hz
         }
     }
 }
+
+// ------------------ //
+// --- UNIT TESTS --- //
+// ------------------ //
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use anyhow::Result;
+    use std::path::PathBuf;
+
+    // Create a fake plugin structure for oar3 plugin
+    fn create_mock_plugin() -> OARPlugin {
+        OARPlugin {
+            config: OAR3Config {
+                path: PathBuf::from("/sys/fs/cgroup/kubepods.slice/"),
+                poll_interval: Duration::from_secs(1),
+            },
+            watcher: None,
+            metrics: None,
+        }
+    }
+
+    // Test default configuration of oar3 plugin
+    #[test]
+    fn test_default_config() {
+        let result = OARPlugin::default_config().unwrap();
+        assert!(result.is_some(), "Expected Some(ConfigTable): result = None");
+
+        let config_table = result.unwrap();
+        let config: OAR3Config = deserialize_config(config_table).expect("Failed to deserialize config");
+
+        assert_eq!(config.path, PathBuf::from("/sys/fs/cgroup/"));
+        assert_eq!(config.poll_interval, Duration::from_secs(1));
+    }
```

Comment:
I know that you want to increase the code coverage, but:
- `serialize_config` and `deserialize_config` are already tested elsewhere
- comparing the values with what `Default::default` returns is just duplicating code IMO
- it could be written more cleanly as `assert_eq!(Some(OAR3Config::default()), OARPlugin::default_config().unwrap())` (and deriving Eq for the config).

In the end, I'm not even convinced it's worth spending time to check that :thinking: